src/main/webapp/app/shared/tab-bar/tab-bar.scss

```diff
@@ -38,6 +39,16 @@
             right: 0;
             transform: translate(50%, -50%);
         }
+
+        @media (max-width: 720px) {
+            :nth-child(2) {
+                display: none;
+            }
```

Comment:
This could technically break when we change the order or add something else. Let's give the elements you want to hide a class and select them explicitly

Reply:
Resolved by adding the class "tab-link-text" :)